src/main/java/com/example/gamelist/Controller.java

```diff
@@ -1,10 +1,23 @@
 package com.example.gamelist;
 
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.naming.Name;
+import java.util.List;
+
 @RestController
 public class Controller {
-    public String gamelist{
-        return
+
+    private final GameMapper gameMapper;
+
+    public Controller (GameMapper gameMapper) {
+        this.gameMapper = gameMapper;
+    }
+
+    @GetMapping("/gamelist")
+    public  List<Name> gamelists(){
```

Comment:
Nameっていうクラス、元々Javaが持ってるクラスみたいなので、きっと使いたいクラスじゃなさそうですね。
```suggestion
    public  List<GameName> gamelists(){
```